src/Jupyter/Visualization/StateDisplayOperations.cs

```diff
@@ -97,13 +98,35 @@ public override bool Dump(IQArray<Qubit>? qubits = null)
             // At this point, _data should be filled with the full state
             // vector, so let's display it, counting on the right display
             // encoder to be there to pack it into a table.
-            Channel.Display(new DisplayableState
+
+            var count = 0; //TODO: figure out how to increment
```

Comment:
Probably easier to just make a new GUID here.
```suggestion
            var id = System.Guid.NewGuid(); //TODO: figure out how to increment
```